lib/Twitch.js

```diff
@@ -72,7 +74,21 @@ export default class TwitchApi {
 
   static async call(url, accessToken) {
     try {
-      const response = await fetch(url, TwitchApi.getFetchOptions(accessToken));
+      let response;
+
+      while (true) {
+        response = await fetch(url, TwitchApi.getFetchOptions(accessToken));
+
+        if (response.status != 429)
```

Comment:
Should be !== in this JavaScript land

Reply:
And we should probably use { } around the if because it's consistent with the rest of the code